native/src/main/java/io/ballerina/lib/ldap/Utils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.lib.ldap;
+
+import io.ballerina.runtime.api.creators.ErrorCreator;
+import io.ballerina.runtime.api.values.BError;
+
+import static io.ballerina.lib.ldap.ModuleUtils.getModule;
+import static io.ballerina.runtime.api.utils.StringUtils.fromString;
+
+public final class Utils {
```

Comment:
missing java docs. Please update all the newly added classes with java docs

Reply:
Fixed in [9773db2](https://github.com/Nuvindu/module-ldap/pull/1/commits/9773db2c06375f79092e796c148454d63fa0a527)